data_contribution_2_data.md

```diff
@@ -0,0 +1,907 @@
+
+When appropriate, archiving your research in Arcus is expected from Scientific Projects with an Arcus Lab. This is documented in the [Arcus Terms of Use](https://arcus.chop.edu/terms-of-use). Archiving is required if you would like to move any data created within an Arcus Lab to a new Scientific Project or if other research teams would like to reuse your data.
+
+When you are ready to archive your lab data, please submit the following request in the [Arcus Help Center](https://pm.arcus.chop.edu/servicedesk/customer/portal/6/create/256) to begin the data contribution process.
+
+## Arcus Data Lifecycle
+
+Arcus' goal for research data management and the project template is to provide tools that are relevant throughout the entire lifecycle of research data. The project template is designed to be adaptable and iterative to capture the wide range of research activities at CHOP. The project template combines flexibility to encompass diverse data capturing needs, while also maintaining a consist structure for all archived data which facilitates seamless communication among various projects spanning different domains, thereby promoting effective data sharing.
+
+**How was this structure developed?**
+
+The CHOP project template file directory structure was adapted from [DrivenData’s Cookiecutter Data Science template](https://cookiecutter.readthedocs.io/en/stable/). It was adapted by former Arcus Digital Archivist, Christiana Dobrzynski, and former CHOP Bioinformaticist, Perry Evans. Both Arcus’s and DrivenData’s templates aim to organize research data and tools for accuracy and reproducibility. See [DataDriven's introduction](http://drivendata.github.io/cookiecutter-data-science/) to learn more about the goals and purpose of project template structures for data preservation and sharing.
+
+The CHOP project template evolved through iterations and feedback from CHOP researchers. A multi-disciplinary groups of practicioners were consulted in the template adaptation development, including:
+
+- Bioinformatics
+- Cancer research
+- Microbiome center
+- Research IT
+- Clinical sequening unit
+- Medical Informatics Unit
+
+**How is the Project Template used at Arcus?**
+
+The Project Template prioritizes streamlined archiving and reproducible research pathways. It archives a wide range of research types from the Research Institute, making them discoverable through tools like [Arcus Cohort Discovery](https://arcus.chop.edu), [Gene](https://chop.alationcloud.com), and [Omics Variant Browser](https://arcus.chop.edu). The Project Template facilitates organizing diverse research data in a single directory structure, enabling automated archiving, metadata management, and data delivery throughout the research data lifecycle. This file directory structure is used for the entire lifecycle of research data within Arcus:
+
+![Graphic representation of the lifecycle of the project template in Arcus](media/project_template/ProjectTemplate_Lifecycle.png)
```

Comment:
Ideally any text on this image (and other images that similarly convey information via text) is captured in alt text or other text elsewhere in the document.  This is important for accessibility, mobile viewing where an image might be v small on a phone, as well as search.  But don't let this hold you back, we can't let the perfect be the enemy of the good!

Reply:
Thank you for pointing this out Joy! We reused graphics we already has for this module, which is why there are so many images with text. We are not going to be able to lay these images out in CSS to match W3 standards. I went through and at least improved all the alt text. We will keep in mind these alt text standards in future.